### PR TITLE
ft #175743403: Added login page

### PR DESCRIPTION
**Pull request by marcelniyi**
### **What does this PR do?**
Implement login page.

### **Description of Task to be completed?**
- Implementation of a login page feature.

- Implementation of responsiveness of a login page.

### **How should this be manually tested?**

 - Clone the repository
 - Do `npm install` to install required modules.
 - Do `npm run start` to start the server.
 - Do `npm run test` to run the tests.
 - Browse the app at `localhost:3000`

### **Any background context you want to provide?**
N/A

### **What are the relevant Pivotal Tracker stories**?
#175743403

### **Screenshots**

Screen Shot 2020-12-06 at 18 26 51

### CODE REVIEW

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]
Could you please consider having the same background for the `Pattern` and the background of the page? because the page background looks somehow having low opacity than the `Pattern` background and leads to showing it extremity.

You also need to write **`tests`** for your page.

**Review by thepatrickniyo — CHANGES_REQUESTED** [review]
You can try to increase the either the height or padding of the login button. to be of the same size with the inputs. Then we be good to go! Anyway most of the required work is done which is good.

**Comment by PatrickNiyogitare28** [comment]
> Could you please consider having the same background for the `Pattern` and the background of the page? because the page background looks somehow having low opacity than the `Pattern` background and leads to showing it extremity.
> 
> You also need to write **`tests`** for your page.

**Review by SamuelSabasNayo — CHANGES_REQUESTED** [review]
As @PatrickNiyogitare28 requested, try to increase the spacing between the Login button, Login with Google, and Login with Facebook. You can consider the Signup page.

File: src/shared/styles/LoginStyles.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+export const Patten = Styled(Pattern)({
+    width: 420,
+     position: 'absolute',
+      margin: 'auto',
+       right: 200,
+        height: 200,
+         backgroundColor: '#fafafa',
+        
+})
```
Comment (SimonImanigirimpuhwe) [inline src/shared/styles/LoginStyles.js]: Good improvement on the pattern

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]
Good job but we still need tests of  **`Login component`**
Consider also resolving conflicting files

**Review by thepatrickniyo — APPROVED** [review]

**Review by SamuelSabasNayo — APPROVED** [review]

**Review by ConstantineMaliza — CHANGES_REQUESTED** [review]
Great work @marcelniyi  You can build login look the same as signup page to avoid the difference look. correct it as @PatrickNiyogitare28 and @SimonImanigirimpuhwe  said

**Pull request merged**